src/components/Picker.js

```diff
@@ -193,6 +193,21 @@ class Picker extends PureComponent {
                                 this.setState({isOpen: false});
                                 this.props.onBlur();
                             },
+
+                            // The following 2 handlers are specific to web (onChange overrides onValueChange on web)
+                            // They are used to make the picker indicator behave the same way as it does on native
```

Comment:
>To solve this, I switched from using onClick to onMouseDown. This solved it simply because of the order the event handlers are called in.

Can we also briefly mention something about this in this code comment? Feels possible someone might make the mistake of switching back to an `onClick` behavior without this.

Reply:
Also, how does `onChange` override `onValueChange` on web?

Reply:
I added some comments regarding this in the `index.js` file of my latest commit.

Let me know if you have any more questions.